app/assets/javascripts/comfy/admin/cms/application.js.coffee

```diff
@@ -1,6 +1,6 @@
 #= require rails-ujs
+#= require comfy/admin/cms/lib/Sortable.min
```

Comment:
Let's not make it feel special. Let's downcase that filename.

Reply:
Done